packages/cli/src/config.ts

```diff
@@ -114,6 +115,28 @@ const colorCategorySchema = z
   })
   .describe('An object with one or more color definitions. The property name is used as the color name.');
 
+const colorModeOverrideSchema = z
+  .object({
+    light: colorSchema.optional(),
+    dark: colorSchema.optional(),
+  })
+  .describe('Override values for semantic color tokens like "background-subtle", "border-default", etc.');
+
+const colorWeightOverrideSchema = z
+  .partialRecord(z.enum(colorNameArr), colorModeOverrideSchema)
+  .describe('The name of the color to add overrides for, e.g. "accent"');
+
+const semanticColorOverrideSchema = z
+  .record(z.string(), colorWeightOverrideSchema)
+  .describe('An object with color names as keys');
+
+const overridesSchema = z
+  .object({
+    colors: semanticColorOverrideSchema.optional(),
+  })
+  .describe('Overrides for generated design tokens')
```

Comment:
perhaps add that only colors are supported currently?

Reply:
```suggestion
  .describe('Overrides for generated design tokens. Only supports colors defined in your theme')
```
Something like this?